Common/utils.ts

```diff
@@ -320,4 +321,26 @@ export function getSeasonWithLeaguePage(seasons: SeasonModel[], seasonId?: strin
     return null;
   }
   return seasons.find(season => season.id.toString() === seasonId.toString() && season.googleSheetParserType !== null && season.googleSheetId !== null);
-}
\ No newline at end of file
+}
+
+export function MakeId(length: number) {
+  let result = '';
+  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  const charactersLength = characters.length;
+  let counter = 0;
+  while (counter < length) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+    counter += 1;
+  }
+  return result;
+}
+
+export function getTeamFromDraftState(game: DraftState, auth: string): 'blueTeam' | 'redTeam' {
+  if (game.blueTeam.auth === auth) {
+    return 'blueTeam';
```

Comment:
Seems like ur using the `blueTeam` and `redTeam` consts more than once, does this need to be defined in a var/type so we only need to make changes in one spot?

Reply:
Tbh probably should be lol